multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/RestartAppStep.java

```diff
@@ -56,6 +56,7 @@ private void setStartupPollingInfo(ProcessContext context) {
 
     private boolean isStarted(CloudControllerClient client, String appName) {
         try {
+            //TODO no need for aux content
```

Comment:
Complete the todo?

Reply:
Oops, these "aux content" todos are leftovers from when I was checking which calls for apps to make 😄 
Will remove them